grails-app/controllers/org/pih/warehouse/api/SelectOptionsApiController.groovy

```diff
@@ -65,7 +65,12 @@ class SelectOptionsApiController {
         List<PaymentTerm> paymentTerms = genericApiService.getList(PaymentTerm.class.simpleName, [sort: "name"]).collect {
             [id: it.id, label: it.name, value: it.id ]
         }
-        render([data: paymentTerms] as JSON)
+        Map blankPaymentTermOption = [
```

Comment:
I think we tend to add an empty option on the front-end side. Maybe would it be good to add an empty option to the select component that could accept the configurable label property?

Reply:
I like this idea, even though this is the only place for now that needs this, this might be very useful for the future.